### ISSUE TITLE
kitchen test fails starting grafana-server

### ISSUE THREAD

**Issue by ghost** [sous-chefs/grafana#61]
I run kitchen test default-ubuntu-1404 and get the following failure:

```
   [2015-07-08T09:28:32+00:00] ERROR: service[grafana-server] (grafana::default line 70) had an error: Mixlib::ShellOut::ShellCommandFailed: Expected process to exit with [0], but received '1'
   ---- Begin output of /etc/init.d/grafana-server start ----
   STDOUT: * Starting Grafana Server
      ...fail!
   STDERR: 
   ---- End output of /etc/init.d/grafana-server start ----
   Ran /etc/init.d/grafana-server start returned 1
   [2015-07-08T09:28:32+00:00] FATAL: Chef::Exceptions::ChildConvergeError: Chef run process exited unsuccessfully (exit code 1)
```

**Comment by lanyonm** [comment]
Hi @DHampe,
This is caused by a race condition in Grafana's startup. I found that it happened on slower hardware, but your mileage may vary.

https://github.com/grafana/grafana/issues/2032
https://github.com/grafana/grafana/issues/1990

We are hoping the new version of Grafana is released soon.  :)

**Comment by tmonk42** [comment]
The fixes for this were slated for Grafana 2.0.3, but it looks like he might be jumping straight to 2.1.
